appveyor/Push-PrefastResultsToAppveyor.ps1

```diff
@@ -0,0 +1,8 @@
+$Dir = get-childitem .\ -Recurse -Filter "*.prefast.sarif"
+
+for ($i=0; $i -lt $Dir.Count; $i++)
+{
+  $fileLocation = $Dir[$i].FullName
+  Push-AppVeyorArtifact $fileLocation
+  .\appveyor\Convert-FileUrisToGitHubUris.ps1 $fileLocation
```

Comment:
Are these two lines in the right order? Shouldn't we convert the file before we ask AppVeyor to save it?